CHANGELOG.md

```diff
@@ -34,6 +34,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - `getMessageGranules()` - Get the granules from a Cumulus execution message, if any.
   - Added `@cumulus/common/cloudwatch-event/isFailedSfStatus()` to determine if a Step Function status from a Cloudwatch event is a failed status
 
+- **CUMULUS-639**
+  - Adds SAML JWT and launchpad token authentication to Cumulus (configurable)
+    - **NOTE** to authenticate with Launchpad ensure your launchpad user_id is in the `<prefix>-UserTable`
```

Comment:
```suggestion
    - **NOTE** to authenticate with Launchpad ensure your launchpad user_id is in the `<prefix>-UsersTable`
```

Reply:
For launchpad token authentication, we check if the user is in the oauth.userGroup, and the user needs to have access to the launchpad workflow (e.g. GSFC-Cumulus-Dev). Keith mentioned they can return the user group information to the SAML response, should we check the user group instead of the table?

Reply:
The SAML response has `userId`, if they returns `user group` information, we can check if the user belongs to a particular group (GSFC-Cumulus-Dev) rather then check the database, the same way as we authorize the launchpad token.
https://github.com/nasa/cumulus/blob/master/packages/common/launchpad.js#L104-L105
https://github.com/nasa/cumulus/blob/master/packages/api/app/launchpadAuth.js#L57

Reply:
I put up a ticket for this.  It will require some work with Launchpad folks. https://bugs.earthdata.nasa.gov/browse/CUMULUS-1578